nginx/hs_proxy/nginx/server_error_on.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<title>HydroShare Unavailable</title>
+<style>
+  body { text-align: center; padding: 150px; }
+  h1 { font-size: 50px; }
+  body { font: 20px Helvetica, sans-serif; color: #333; }
+  article { display: block; text-align: left; width: 650px; margin: 0 auto; }
+  a { color: #dc8100; text-decoration: none; }
+  a:hover { color: #333; text-decoration: none; }
+</style>
+
+<article>
+    <h1>HydroShare Unavailable</h1>
+    <div>
+        <p>Sorry, there is an unexpected problem with HydroShare. The systems operation team has been notified and will address the problem as soon as possible. Sorry for the inconvenience. Contact <a href="mailto:help@cuahsi.org">help@cuahsi.org</a> if you need further information.</p>
```

Comment:
When this message shows, there are no notifications being sent out to the operations team that I am aware of.

Reply:
In most cases, wouldn't Django already send an email to ops team for a 503 error?
https://docs.djangoproject.com/en/1.10/howto/error-reporting/#server-errors